build/target-repository/README.md

```diff
@@ -46,16 +46,16 @@ use Rector\Config\RectorConfig;
 use Rector\Set\ValueObject\SetList;
 use Rector\TypeDeclaration\Rector\Property\TypedPropertyFromStrictConstructorRector;
 
-return static function (RectorConfig $rectorConfig): void {
+return RectorConfig::configure()
     // register single rule
-    $rectorConfig->rule(TypedPropertyFromStrictConstructorRector::class);
-
+    ->withRules([
+        TypedPropertyFromStrictConstructorRector::class
+    ])
     // here we can define, what sets of rules will be applied
     // tip: use "SetList" class to autocomplete sets with your IDE
-    $rectorConfig->sets([
+    ->withSets([
```

Comment:
there is `withPreparedSets` for codeQuality, ref

https://github.com/rectorphp/rector-src/blob/827992ae9784abb4a2a5f5f3aecbe52606f3c737/rector.php#L14-L16

Reply:
updated

Reply:
Hmm, github diff seems not updated, probably a github glicth lately? as I got repeated same notification

Reply:
Is it OK now?